tests/helpers/fixtures.py

```diff
@@ -673,13 +673,21 @@ def download_stub(win_registry, tmpdir, stubs):
 
 
 @pytest.fixture
-def web_history(fake_save_manager, tmpdir, data_tmpdir, config_stub, stubs,
+def database(data_tmpdir, request):
+    """Create a Database object."""
+    name = request.node.get_closest_marker("db_name") or "history.db"
```

Comment:
I don't think this will work - `get_closest_marker` gives you a [Mark object](https://docs.pytest.org/en/latest/reference/reference.html#mark), so you'd need to do something like `.args[0]` on it. However, since we don't actually need to customize the name anywhere, I'd say let's just remove this?

As for the default name, I'd probably call it `test.db` just so that the file is clearly identifiable.

Reply:
Ah, yes, I copied that from the example you provided but never got to test it as we don't really need it. I just forgot to remove it.

I also updated the database name.